server/src/utils/http.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+const httpInstance = axios.create({
+    baseURL: 'https://mastodon.social/api/v1',
```

Comment:
One problem I noticed during my refactoring is that the server name is hardcoded to be mastodon.social. I guess it could work because mastodon.social is the most popular server and ususally has data linked with other servers, but I am not sure if there is any edge case.
Check my PR to see how I made the server name to be part of the Mastodon API (Adaptor)